File: scenarios/multi_step_agent/fixtures/normalize_evidence.py
from __future__ import annotations

import json
import os


def main() -> None:
    evidence = json.loads(os.environ["EVIDENCE_JSON"])
    normalized = {
        "items": [item["value"] for item in evidence["items"]],
        "count": len(evidence["items"]),
        "status": "normalized"
    }
    print(json.dumps(normalized))


if __name__ == "__main__":
    main()

File: scenarios/multi_step_agent/fixtures/normalize_evidence.ts
const evidence = JSON.parse(process.env.EVIDENCE_JSON ?? "{}") as { items: Array<{ value: string }> };
const normalized = {
  items: evidence.items.map((item) => item.value),
  count: evidence.items.length,
  status: "normalized",
};

console.log(JSON.stringify(normalized));
